woocommerce/woocommerce-gateway-stripe
# Fix E2E Woo setup admin username

## Changes proposed in this Pull Request:

We were using a generic admin username for the setup steps in the E2E setup. In JN sites, the admin user is usually `demo`, this was causing some setup steps to don't work.

## Testing instructions

1. Run the E2E setup in a fresh JN site.
2. Ensure there are shipping methods in the test site after the setup is finished. The shipping zone `Everywhere` should have the shipping methods `Flat rate` and `Free shipping`. `/wp-admin/admin.php?page=wc-settings&tab=shipping`

---


**Post merge**
